### ISSUE TITLE
type names

### ISSUE THREAD

**Issue by Alexey-T** [eriklins/Pascal-Bindings-For-SimpleBLE-Library#3]
```
  simpleble_manufacturer_data_t = record
  simpleble_adapter_t = size_t;
  simpleble_peripheral_t = size_t;

  simpleble_os_t 
```
better rename to Pascal style!
ie

- TSimpleBleManufacturerData
- TSimpleBleAdapter
- TSimpleBlePeripheral
- etc

**Comment by eriklins** [comment]
That's true. I wanted to keep it close to the original header files in the first place, which made testing and porting of the examples more simple. But yes, I will do this and follow Pascal style. Thanks!

**Comment by eriklins** [comment]
Fixed with commit 087cbea4e23d1d6fb673ddddf305e1a343c4ca1d
